docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -202,11 +202,6 @@ Refer to the documentation of the respective package for such dependencies and t
 Common Platform Issues
```

Comment:
What about s/Platform/System here? That might help further disambiguate with the enterprise bits.